feature/gnmi/subscribe/tests/gnmi_sample_mode_test/gnmi_sample_mode_test.go

```diff
@@ -37,11 +37,11 @@ func TestGNMISampleMode(t *testing.T) {
 
 	desc := p1Stream.Next()
 	if desc == nil {
-		t.Errorf("Interface %q telemetry not received before config", p1.Name())
+		t.Logf("Interface %q telemetry not received before config", p1.Name())
 	} else {
 		v, ok := desc.Val()
 		if !ok {
-			t.Errorf("Interface %q telemetry empty before config", p1.Name())
+			t.Logf("Interface %q telemetry empty before config", p1.Name())
 		}
 		t.Logf("Description before config: %s", v)
 	}
```

Comment:
this whole block (lines 38-47) is now a no-op and can be removed

Reply:
Checked in changes as per the comment.